charts/loki-distributed/README.md

```diff
@@ -253,11 +253,9 @@ kubectl delete statefulset RELEASE_NAME-loki-distributed-querier -n LOKI_NAMESPA
 | ingester.livenessProbe | object | `{}` | liveness probe settings for ingester pods. If empty use `loki.livenessProbe` |
 | ingester.maxUnavailable | string | `nil` | Pod Disruption Budget maxUnavailable |
 | ingester.nodeSelector | object | `{}` | Node selector for ingester pods |
-| ingester.persistence.annotations | object | `{}` | Annotations for ingester PVCs |
+| ingester.persistence.claims | string | `nil` |  |
```

Comment:
`ingester.persistence.claims` type should be `list`.

Reply:
updated